src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -459,16 +459,18 @@ public List<URL> listURLs(String context) throws IOException {
   @Override
   public ComponentInfo getComponentInfo(String project, String component, String path,
                                         ComponentKind kind) throws IOException {
+    boolean isPodmanPresent = !execute(command, envVars, "version").contains("unable to fetch the podman client version");
```

Comment:
podman check is now here, added as componentInfo status. This is done when loading tree component where the debug podman action resisdes.